cdap-ui/app/cdap/components/If/index.tsx

```diff
@@ -14,15 +14,34 @@
  * the License.
  */
 import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import withStyles from '@material-ui/core/styles/withStyles';
 
 interface IIfComponentProps {
   condition: boolean;
   children: React.ReactNode;
+  invisible?: boolean;
+  HiddenContainer?: React.ReactNode;
 }
 
-const If: React.FC<IIfComponentProps> = ({ condition, children }) => {
+const If: React.FC<IIfComponentProps> = ({
+  condition,
+  children,
+  invisible = false,
```

Comment:
Do we want to add this to `If` or create a new `Hidden` component? `If` seems like it should stay a one-purpose component.

Reply:
The thought process was if the condition is false the children under the `If` doesn't get rendered. We can choose to make it invisible (occupying the space) or set it to display none there by avoiding browser to paint anything. 

If you feel we should have this as a separate component I can make it but it will still do the same thing done by `If` except set the visibility to hidden instead of display none.